cmd2/argparse_completer.py

```diff
@@ -681,20 +698,26 @@ def _complete_arg(
         # Check if the argument uses a specific tab completion function to provide its choices
         if isinstance(arg_choices, ChoicesCallable) and arg_choices.is_completer:
             args.extend([text, line, begidx, endidx])
-            results = arg_choices.to_call(*args, **kwargs)
+            results = arg_choices.to_call(*args, **kwargs)  # type: ignore[arg-type]
 
         # Otherwise use basic_complete on the choices
         else:
             # Check if the choices come from a function
-            if isinstance(arg_choices, ChoicesCallable) and not arg_choices.is_completer:
-                arg_choices = arg_choices.to_call(*args, **kwargs)
+            completion_items: List[str]
+            if isinstance(arg_choices, ChoicesCallable):
+                if not arg_choices.is_completer:
+                    completion_items = arg_choices.to_call(*args, **kwargs)  # type: ignore[arg-type]
+                else:
+                    completion_items = []
```

Comment:
Consider adding a unit test for this new `else` case(s)

Reply:
This is an impossible case. I'm taking the line out.